src/sctp/mod.rs

```diff
@@ -369,6 +376,57 @@ impl RtcSctp {
         Ok(stream.write_with_ppi(buf, ppi)?)
     }
 
+    pub fn buffered_amount(&mut self, id: u16) -> Result<usize, SctpError> {
+        if self.state != RtcSctpState::Established {
+            return Err(SctpError::BufferedAmountQueryBeforeEstablished);
+        }
+
+        let assoc = self
+            .assoc
+            .as_mut()
+            .ok_or(SctpError::BufferedAmountQueryBeforeEstablished)?;
+
+        let rec = self
+            .entries
+            .iter()
+            .find(|e| e.id == id)
+            .expect("stream entry for write");
+
+        if rec.state != StreamEntryState::Open {
+            return Err(SctpError::BufferedAmountQueryBeforeEstablished);
+        }
+
+        let stream = assoc.stream(id)?;
+
+        Ok(stream.buffered_amount()?)
+    }
+
+    pub fn set_buffered_amount_low_threshold(&mut self, id: u16, threshold: usize) -> Result<(), SctpError> {
+        if self.state != RtcSctpState::Established {
+            return Err(SctpError::BufferedAmountQueryBeforeEstablished);
+        }
+
+        let assoc = self
+            .assoc
+            .as_mut()
+            .ok_or(SctpError::BufferedAmountQueryBeforeEstablished)?;
+
+        let rec = self
+            .entries
+            .iter()
+            .find(|e| e.id == id)
+            .expect("stream entry for write");
+
+        if rec.state != StreamEntryState::Open {
+            return Err(SctpError::BufferedAmountQueryBeforeEstablished);
+        }
+
+        let mut stream = assoc.stream(id)?;
+        stream.set_buffered_amount_low_threshold(threshold)?;
+
+        Ok(())
+    }
+
```

Comment:
By using `ChannelConfig`, we can simplify this since I don't think we need to consider the "Before Established" situation. The `Channel::buffered_amount()` could return `0` until the SCTP is established.